project/src/test/java/caselab/service/users/UserMapperTest.java

```diff
@@ -0,0 +1,126 @@
+package caselab.service.users;
+
+import caselab.controller.document.payload.DocumentResponse;
+import caselab.controller.users.payload.UserResponse;
+import caselab.controller.users.payload.UserUpdateRequest;
+import caselab.domain.entity.ApplicationUser;
+import caselab.domain.entity.Document;
+import caselab.service.document.DocumentMapper;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mapstruct.factory.Mappers;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.openMocks;
+
+public class UserMapperTest {
+    @InjectMocks
+    private UserMapper userMapper = Mappers.getMapper(UserMapper.class);
+
+    @Mock
+    private DocumentMapper documentMapper;
+
+    @BeforeEach
+    void setUp() {
+        openMocks(this);
+    }
+
+    @Test
+    void entityToResponse_shouldConvertUserToUserResponse() {
+        ApplicationUser user = ApplicationUser.builder().id(1L).login("testUser").displayName("Test User").build();
```

Comment:
В билдере каждый метод должен быть с новой строки
```suggestion
        ApplicationUser user = ApplicationUser.builder()
                                                                   .id(1L)
                                                                   .login("testUser")
                                                                   .displayName("Test User")
                                                                   .build();
```